README.adoc

```diff
@@ -24,7 +24,7 @@ To use or develop FPP in a standalone mode, you should install these tools.
 To use FPP as part of F Prime development, you can install the F Prime tool set,
 which includes FPP.
 See the F Prime installation instructions
-https://github.com/nasa/fprime/blob/master/docs/INSTALL.md[here].
+https://fprime.jpl.nasa.gov/latest/getting-started/installing-fprime/[here].
```

Comment:
This link is giving me a 404.

Reply:
This link works: https://fprime.jpl.nasa.gov/latest/docs/getting-started/installing-fprime/

Reply:
Good catch, thanks! Fixed.